specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/AddressesGetAtResourceGrouplevel.json

```diff
@@ -0,0 +1,108 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/addresses?api-version=2020-12-01-preview"
+    ]
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "shippingAddress": {
+                "streetAddress1": "16 TOWNSEND ST",
+                "streetAddress2": "UNIT 1",
+                "streetAddress3": "",
+                "city": "San Francisco",
+                "stateOrProvince": "CA",
+                "country": "US",
+                "postalCode": "94107",
+                "zipExtendedCode": "",
+                "companyName": "Microsoft",
+                "addressType": "None"
+              },
+              "contactDetails": {
+                "contactName": "Petr Cech",
+                "phone": "123344334",
+                "phoneExtension": ""
+              }
+            },
+            "location": "westus",
```

Comment:
>westus [](start = 25, length = 6)

Does location signifies anything for the address object?

Reply:
yeah location where address resource is created.